short videos, quizzes rephrase, added questions to all lessons and rep… (#416)
neo4j-graphacademy/courses: asciidoc/courses/aura-fundamentals/modules/3-services-tools/lessons/1-import/lesson.adoc

image::images/01_data_importer_model_person.jpg[importer_person,width=600,align=center]

You can edit the properties of the nodes by clicking on the pencil icon next to each property: 

video::https://cdn.graphacademy.neo4j.com/courses/aura-fundamentals/edit-fields.mp4[role="cdn",  Edit Fields, 350, 300]

Create and define the ACTED_IN relationship.

* Drag the relationship from the Person node to the Movie node using the plus-sign when you hover over the edge of the Person node.
* Name the relationship type as **ACTED_IN**.
* Node ID mapping should be automatic.
* Click on **Map from table** (csv file in this case), **characters**.

image::images/01_data_importer_model_actedin.jpg[importer_actedin,width=600,align=center]

Note the green checkmark that indicates that an element has been mapped correctly.

== Analyse and confirm model parameters

Before Importing, you can analyse and edit the model parameters, as well as confirming the primary key of the nodes. 

video::https://cdn.graphacademy.neo4j.com/courses/aura-fundamentals/load-data-params.mp4[role="cdn", Load Data Parameters, 350, 300]

== Import

> To proceed with data ingestion,**Run import** is the next step, and it needs a connection to the database.

Reviewer: The language feels a little disjointed and difficult to read.

```suggestion
Use **Run import"" to ingest the data. You will need to connect to the database.
```
